assets/css/main.css

```diff
@@ -2791,7 +2791,7 @@
 			font-size: 0.9em;
 			font-weight: 400;
 			padding: 0 0.75em 0.75em 0.75em;
-			text-align: left;
+			text-align: center;
```

Comment:
Any reason for this design change?

Reply:
The table did not fit on the page right for some reason with left text-align.